tsdb/wal/checkpoint.go

```diff
@@ -270,7 +271,11 @@ func Checkpoint(logger log.Logger, w *WAL, from, to int, keep func(id uint64) bo
 	return stats, nil
 }
 
-func checkpointDir(dir string, i int) string {
+func CheckpointSize(dir string) (int64, error) {
+	return fileutil.DirSize(dir)
+}
```

Comment:
Why do we need this method if all we are doing is fileutil.DirSize?

Reply:
Removed!